src/types/shared.ts

```diff
@@ -64,17 +65,23 @@ export interface IDataType {
 export interface IInstance {
   appId: string;
   created?: string;
+  createdBy?: string;
   data: IData[];
   dueBefore?: string;
   id: string;
   instanceOwner: IInstanceOwner;
   instanceState?: IInstanceState;
   lastChanged?: string;
+  lastChangedBy?: string;
   org: string;
   selfLinks?: ISelfLinks | null;
   status?: IInstanceStatus | null;
   title?: ITitle | null;
   visibleAfter?: string;
+  process?: IProcess;
```

Comment:
Isn't this just deleted by `cleanUpInstanceData()`? Might be best not to trick developers into thinking the process data can be gleaned from the instance data, as the dedicated process data query fetches more stuff not returned in the instance data itself.